docs/src/modules/java/pages/quickstart/cr-value-entity-java.adoc

```diff
@@ -7,7 +7,7 @@ Learn how to create a customer registry in Java, package it into a container, an
 
 == Before you begin
 
-* If you're new to Kalix, {console}[create an account, window="console"] so you can try out Kalix for free.
+* If you're new to Kalix, {console}[create an account, window="console"], so you can try out Kalix for free.
```

Comment:
```suggestion
* If you're new to Kalix, {console}[create an account, window="console"] so you can try out Kalix for free.
```

I think in this case it shouldn't have a `comma` actually... TLDR: if you can replace "so" with "so that" it means it should not have a comma before "so".

See section "Comma before so" [here](https://www.grammarly.com/blog/how-to-use-commas-in-your-writing/?gclid=Cj0KCQjwteOaBhDuARIsADBqRegAGnCQbF5WpdLRiNp3p2SvwZK5Hy7-DiP0B5vX6UckXfgA8EFHOnEaAjzSEALw_wcB&gclsrc=aw.ds).

Reply:
I added it because IntelliJ was generating a warning. I will remove my change and disable the check in IntelliJ. It seems that it can't be trusted. 😄